packages/frontend/components/basic/BasicInput.vue

```diff
@@ -0,0 +1,35 @@
+<script setup lang="ts">
+interface Props {
+  modelValue?: string
+  placeholder: string
+  label?: string
+  type?: string
```

Comment:
Type should be more strict, only 'input', 'password' and 'agree' are supported.

Agree is not a valid input type, there is a checkbox for that. This component should solely be a text / password input, checkbox should be its own component.

Reply:
fuck me i thought i'd get some more time off

Reply:
wsid